### PR TITLE
Pin `prompt_toolkit` in travis setup

### PR DESCRIPTION
**Pull request by gforsyth**
Temporary fix (that we implemented everywhere else) to prevent failures
due to the prompt_toolkit release

No news item required for this one

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ install:
   - if [[ $BUILD_DOCS = true ]]; then
       python setup.py install;
       pip install -r requirements-docs.txt;
-      pip install pygments prompt_toolkit ply psutil ipykernel matplotlib;
+      pip install pygments prompt_toolkit<=2 ply psutil ipykernel matplotlib;
```
Comment (scopatz) [inline .travis.yml]: Did you mean `<2`

**Comment by scopatz** [comment]
Thanks for putting this in @gforsyth!

File: .travis.yml (hunk 1 above)
Comment (gforsyth) [reply .travis.yml]: Who are you who is so wise in the ways of operators? :headdesk:

Comment (scopatz) [reply .travis.yml]: I'm an enchanter.

05173982650c1f7883196880e703cba1a97a53-wm

**Pull request merged**